core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -1899,7 +1899,7 @@ class SparkContext(config: SparkConf) extends Logging {
     if (path == null || path.isEmpty) {
       logWarning("null or empty path specified as parameter to addJar")
     } else {
-      val key = if (path.contains("\\")) {
+      val key = if (path.contains("\\") && Utils.isWindows) {
```

Comment:
@AngersZhuuuu . This is `CORE` instead of `SQL`.
- `core/src/main/scala/org/apache/spark/SparkContext.scala`

Reply:
> @AngersZhuuuu . This is `CORE` instead of `SQL`.
> 
> * `core/src/main/scala/org/apache/spark/SparkContext.scala`

Sorry, since always work at `SQL`  module